### DISCUSSION TITLE
Troubleshooting issue #27

### DISCUSSION THREAD

**Discussion by jsaddiction** [jsaddiction/TrailerTech#30] [Q&A]
@bozrdang Lets further discuss #27 here

**Comment by bozrdang** [comment]
OK

**Reply by jsaddiction** [reply]
I think something is hanging in your install methodology.  In the linuxserver.io image they run all scripts contained within the /config/custom-cont-init.d path.  In your compose file the /config path should be mapped to a place in your storage environment to persist that data.  This /config path is also where things like your radarr db files are kept.  Anyway if you create the Install_TrailerTech.sh file within the /config/custom-cont-init.d/ path it will run when the container is created not restarted.

**Reply by bozrdang** [reply]
I run all my Docker containers in Unraid.  And while you can use a compose file most are setup with a template.  That's the only way I've ever created a container.  The /config path is persistent and mapped to a physical SSD.  I installed the Linuxserver.io Radarr container and started it.  I then stopped and removed it, created the /config/custom-cont-init.d folder, created the Install_TrailerTech.sh file there and re-created the container.  After starting it the /config/scripts/TrailerTech folder was created and filled with files.

**Reply by jsaddiction** [reply]
once it was filled with files did you see an error running the application?

**Reply by jsaddiction** [reply]
also checkout the "one liner" command that should make things a little easier.  It can be issued directly from your CLI outside of the container.

**Reply by bozrdang** [reply]
> once it was filled with files did you see an error running the application?

Yes.  The same error I got on hotio's container.

I'll check out the one liner.

**Reply by jsaddiction** [reply]
This issue may be something peculiar going on with unraid and its ecosystem.  I'm not sure how to resolve this.  My test bed is a dell R720 running Ubuntu 20 and docker.

**Reply by bozrdang** [reply]
Maybe.  Your fix worked for me in my hotio container so I'm good.  I just tried the Linuxserver.io container as a test.
